.eslintrc

```diff
@@ -29,8 +29,17 @@
 
     "no-console": "off",
     "eqeqeq": ["error", "always", { "null": "ignore" }],
-    "strict": ["error", "global"]
+    "strict": ["error", "global"],
+    "promise/no-native": "error"
   },
+  "overrides": [
```

Comment:
Adds `promise/no-native` lint requirement and ignores test folders.

Reply:
very nice! Can't you use glob syntax to ignore a single pattern:
```
"files": ["test/**/*.js"]
```